src/Main.hs

```diff
@@ -53,6 +60,26 @@ createTables :: Connection -> IO ()
 createTables db = do
   execute_ db "CREATE TABLE IF NOT EXISTS sessions (token VARCHAR(40) PRIMARY KEY, userId INTEGER NOT NULL, timestamp INTEGER NOT NULL);"
   execute_ db "CREATE TABLE IF NOT EXISTS users (userId INTEGER PRIMARY KEY AUTOINCREMENT, username VARCHAR(40) NOT NULL, passwordHash VARCHAR(80) NOT NULL);"
+  {-
+    Recipe:
+      id:           unique id for identification of recipes
+      title:        Name of the recipe (40 chars max)
+      description:  Description of the recipe
+      ingredients:  Comma-seperated list of ingredients stored as string (in Haskell format)
+      timeMinutes:  time in minutes
+      costCents:    cost in cents
+      stars:        integer between 0 and 5
+  -}
+  execute_
+    db
+    "CREATE TABLE IF NOT EXISTS recipes (\
+    \ id INTEGER PRIMARY KEY AUTOINCREMENT,\
```

Comment:
vlt `recipeId` nennen

Reply:
Renamed in 9d8ac164eaaf6e8355e1254ae071ac5eb29b6aef